app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -147,7 +152,10 @@ export const ListingDetailsEligibility = ({
                   <p>
                     {renderInlineMarkup(
                       t("listings.customListingType.educator.eligibility.part2", {
-                        chisholmLink: "https://sf.gov/apply-shirley-chisholm-village-housing",
+                        chisholmLink: getSfGovUrl(
+                          "https://sf.gov/apply-shirley-chisholm-village-housing",
+                          10543
+                        ),
```

Comment:
[Below](https://github.com/SFDigitalServices/sf-dahlia-web/blob/DAH-1392-sfgov-links/app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx#L204) there is another reference to this same `chisholmLink`